src/themes/shared/borders.ts

```diff
@@ -1,5 +1,6 @@
 export const borders = {
   none: 0,
   '1px': '1px solid',
+  '2px': '1px solid',
```

Comment:
Mmmm  I think you need `2px`

```suggestion
  '2px': '2px solid',
```

Reply:
lol, thanks :)